docs/user-guide/api/classifiers.rst

```diff
@@ -12,4 +12,4 @@ Classifiers
     :members:
 
 .. autoclass:: nemo_curator.classifiers.AegisClassifier
-    :members:
\ No newline at end of file
+    :members:
```

Comment:
Do you know what the change was here?

Reply:
I think this is just adding an empty line to the bottom of the file (there previously wasn't one).